### DISCUSSION TITLE
54.1 Join (dependencies)

### DISCUSSION THREAD

**Discussion by njr0** [google/comprehensive-rust#1317] [General]
I had to add these dependencies to may `Cargo.toml` to make it work. As with the last few suggestions, I think it will help to includes these in the speaker notes or main text, especially since this can't be run in the playground. (And who wants to be in a playground anyway?)

```
[dependencies]
tokio = { version = "1.32.0", features = ["full"] }
reqwest = { version = "0.11.22", features = ["blocking", "rustls-tls"] }
futures = "0.3"
anyhow = "1.0"
```

**Comment by mgeisler** [comment]
Yeah, we should be more careful about the dependencies needed here! I created #1330 for this.
